projects/1.18/assets/create-connected-block-textures/create_connected_blocks/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+  "block.create_connected_blocks.lapis_clear_glass": "青金石通透玻璃",
+  "block.create_connected_blocks.lapis_glass": "青金石玻璃",
+  "block.create_connected_blocks.light_blue_railway_casing": "淡蓝色火车外壳",
+  "block.create_connected_blocks.light_grey_railway_casing": "淡灰色火车外壳",
+  "block.create_connected_blocks.lime_railway_casing": "黄绿色火车外壳",
+  "block.create_connected_blocks.magenta_railway_casing": "品红色火车外壳",
+  "block.create_connected_blocks.netherite_block": "连接下界合金块",
+  "block.create_connected_blocks.netherite_clear_glass": "下界合金通透玻璃",
+  "block.create_connected_blocks.netherite_glass": "下界合金玻璃",
+  "block.create_connected_blocks.orange_railway_casing": "橙色火车外壳",
+  "block.create_connected_blocks.pink_railway_casing": "粉红色火车外壳",
+  "block.create_connected_blocks.purple_railway_casing": "紫色火车外壳",
+  "block.create_connected_blocks.red_railway_casing": "红色火车外壳",
+  "block.create_connected_blocks.reinforced_brass_block": "加固黄铜块",
+  "block.create_connected_blocks.reinforced_copper_block": "加固铜块",
+  "block.create_connected_blocks.reinforced_diamond_block": "加固钻石块",
+  "block.create_connected_blocks.reinforced_emerald_block": "加固绿宝石块",
+  "block.create_connected_blocks.reinforced_gold_block": "加固金块",
+  "block.create_connected_blocks.reinforced_iron_block": "加固铁块",
+  "block.create_connected_blocks.reinforced_lapis_block": "加固青金石块",
+  "block.create_connected_blocks.reinforced_netherite_block": "加固下界合金块",
+  "block.create_connected_blocks.reinforced_zinc_block": "加固锌块",
+  "block.create_connected_blocks.sea_lantern": "连接海晶灯",
+  "block.create_connected_blocks.stone": "连接光滑石",
+  "block.create_connected_blocks.white_railway_casing": "白色火车外壳",
+  "block.create_connected_blocks.yellow_railway_casing": "黄色火车外壳",
+  "block.create_connected_blocks.zinc_block": "连接锌块",
+  "block.create_connected_blocks.zinc_clear_glass": "锌质通透玻璃",
+  "block.create_connected_blocks.zinc_glass": "锌玻璃",
+  "item.create_connected_blocks.brass_reinforcement": "黄铜加固材",
+  "item.create_connected_blocks.copper_reinforcement": "铜加固材",
+  "item.create_connected_blocks.diamond_reinforcement": "钻石加固材",
+  "item.create_connected_blocks.emerald_reinforcement": "绿宝石加固材",
+  "item.create_connected_blocks.golden_reinforcement": "黄金加固材",
+  "item.create_connected_blocks.incomplete_brass_reinforcement": "不完整的黄铜钢筋",
+  "item.create_connected_blocks.incomplete_copper_reinforcement": "不完整的铜加固材",
+  "item.create_connected_blocks.incomplete_diamond_reinforcement": "不完整的钻石加固材",
+  "item.create_connected_blocks.incomplete_emerald_reinforcement": "不完整的绿宝石加固材",
+  "item.create_connected_blocks.incomplete_golden_reinforcement": "不完整的黄金加固材",
+  "item.create_connected_blocks.incomplete_iron_reinforcement": "不完整的铁加固材",
+  "item.create_connected_blocks.incomplete_lapis_reinforcement": "不完整的青金石加固材",
+  "item.create_connected_blocks.incomplete_netherite_reinforcement": "不完整的下界合金钢筋",
+  "item.create_connected_blocks.incomplete_zinc_reinforcement": "不完整的锌加固",

```

```suggestion
  "item.create_connected_blocks.incomplete_brass_reinforcement": "黄铜加固材（半成品）",
  "item.create_connected_blocks.incomplete_copper_reinforcement": "铜加固材（半成品）",
  "item.create_connected_blocks.incomplete_diamond_reinforcement": "钻石加固材（半成品）",
  "item.create_connected_blocks.incomplete_emerald_reinforcement": "绿宝石加固材（半成品）",
  "item.create_connected_blocks.incomplete_golden_reinforcement": "黄金加固材（半成品）",
  "item.create_connected_blocks.incomplete_iron_reinforcement": "铁加固材（半成品）",
  "item.create_connected_blocks.incomplete_lapis_reinforcement": "青金石加固材（半成品）",
  "item.create_connected_blocks.incomplete_netherite_reinforcement": "下界合金加固材（半成品）",
  "item.create_connected_blocks.incomplete_zinc_reinforcement": "锌加固材（半成品）",
```
请仔细对照create本身的语言文件以及做好上下文统一